### PR TITLE
CAT-963: add riskprofiler certs

### PR DESCRIPTION
**Pull request by kkonuganti**

### CODE REVIEW

File: namespaces/live.cloud-platform.service.justice.gov.uk/offender-categorisation-dev/06-certificates.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+metadata:
+  name: dev.offender-risk-profiler.service.justice.gov.uk
+  namespace: offender-categorisation-dev
+spec:
+  secretName: offender-risk-profiler-cert
+  issuerRef:
+    name: letsencrypt-production
+    kind: ClusterIssuer
+  dnsNames:
+  - dev.offender-risk-profiler.service.justice.gov.uk
```
Comment (Mjwillis) [inline namespaces/live.cloud-platform.service.justice.gov.uk/offender-categorisation-dev/06-certificates.yml:12]: These DNS don't follow convention, should be `offender-risk-profiler-dev.hmpps.service.justice.gov.uk`

File: namespaces/live.cloud-platform.service.justice.gov.uk/offender-categorisation-preprod/06-certificates.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+metadata:
+  name: preprod.offender-risk-profiler.service.justice.gov.uk
+  namespace: offender-categorisation-dev
+spec:
+  secretName: offender-risk-profiler-cert
+  issuerRef:
+    name: letsencrypt-production
+    kind: ClusterIssuer
+  dnsNames:
+  - preprod.offender-risk-profiler.service.justice.gov.uk
```
Comment (Mjwillis) [inline namespaces/live.cloud-platform.service.justice.gov.uk/offender-categorisation-preprod/06-certificates.yml:12]: ditto These DNS don't follow convention, should be `offender-risk-profiler-preprod.hmpps.service.justice.gov.uk`

File: namespaces/live.cloud-platform.service.justice.gov.uk/offender-categorisation-prod/06-certificates.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+metadata:
+  name: offender-risk-profiler.service.justice.gov.uk
+  namespace: offender-categorisation-dev
+spec:
+  secretName: offender-risk-profiler-cert
+  issuerRef:
+    name: letsencrypt-production
+    kind: ClusterIssuer
+  dnsNames:
+  - offender-risk-profiler.service.justice.gov.uk
```
Comment (Mjwillis) [inline namespaces/live.cloud-platform.service.justice.gov.uk/offender-categorisation-prod/06-certificates.yml:12]: ditto These DNS don't follow convention, should be `offender-risk-profiler.hmpps.service.justice.gov.uk`

**Review by Mjwillis — CHANGES_REQUESTED** [review]
Can we change these to be consistent. Was there a reason for exposing the ingress in the end and not use the healthkick?

File: namespaces/live.cloud-platform.service.justice.gov.uk/offender-categorisation-prod/06-certificates.yml (hunk 3 above)
Comment (kkonuganti) [reply namespaces/live.cloud-platform.service.justice.gov.uk/offender-categorisation-prod/06-certificates.yml:12]: thanks will change them

**Review by vijay-veeranki — APPROVED** [review]

**Review by vijay-veeranki — APPROVED** [review]

**Review by Mjwillis — APPROVED** [review]
LGTM

**Comment by kkonuganti** [comment]
> 

we might use this for other apis going forward.

**Pull request merged**